[modelica] Follow-up fixes for Modelica language module

atrosinenko opened the pull request:
Follow-up fixes for #2041.

pmd-test commented:
<!--
  0 Errors
  0 Warnings
  1 Message: No java rules are changed!
  0 Markdowns
-->
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="true">No java rules are changed!</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

atrosinenko commented:
OK, I will squash some simple examples into this commit several minutes later.

atrosinenko commented:
Hmm, technically, I have added examples. But when testing an example for `AmbiguousResolution` rule, OpenModelica originally refused to compile it not due to ambiguity, but because it can not find `Inc1` and `Inc2`, so I just changed them to `Test.Inc1.*` and `Test.Inc2.*`. I also fixed one test description.

So, now rules **looks** better but should probably be fine-tuned by someone with deeper Modelica knowledge later (adding new rules is pending too, of course).
